app/questionnaire/dependencies.py

```diff
@@ -15,35 +15,32 @@
     from app.questionnaire.path_finder import PathFinder  # pragma: no cover
 
 
-def get_block_ids_for_calculated_summary_dependencies(
-    schema: QuestionnaireSchema,
+def map_section_ids_to_routing_path(
```

Comment:
The naming implies it either doesn't return anything and that it mutates something but it doesn't mutate but also returns. The name some reflect the body, so dependencies are an important part. I suggest something like `get_routing_path_block_ids_by_section_for_dependent_sections`. Open to other suggestions.

Reply:
New name suggestion sounds clearer to me - Updated to  `get_routing_path_block_ids_by_section_for_dependent_sections`